coder_dojo_common/models.py

```diff
@@ -0,0 +1,72 @@
+"""Data structures/models"""
+
+from __future__ import annotations
```

Comment:
I'm wondering if these should be in a kenney_assets package within coder_dojo_common - the cli, kenney_assets and models.py?